pkg/mhooks/event.go

```diff
@@ -85,6 +85,10 @@ func ParseEvent(r *http.Request, secret string) (*Event, error) {
 		eventData = &event.transferCreated
 	case EventTypeTransferUpdated:
 		eventData = &event.transferUpdated
+	case EventTypeWalletCreated:
+		eventData = &event.walletCreated
+	case EventTypeWalletUpdated:
+		eventData = &event.walletUpdated
```

Comment:
Need to add the transformers as well below

Reply:
done